Frontend/src/components/forms/ProjectForm.jsx

```diff
@@ -209,12 +209,12 @@ export default function ProjectForm({
     setName(project?.name);
     setCategory(project?.category);
     setType(
-      project?.type.charAt(0) + project?.type.slice(1).toLowerCase() ||
+      project?.type?.charAt(0) + project?.type?.slice(1).toLowerCase() ||
         projectTypes[0].value
     );
     setStartDate(project ? moment(project.startDate) : moment());
     setEndDate(project ? moment(project.endDate) : moment().add(6, "months"));
-    setFRRespID(project?.frresp.id);
+    setFRRespID(project?.frresp?.id);
```

Comment:
Ovdje ista stvar, ne moze se dogoditi da nema frresp (jer je required) tako da upitnik nema svrhu.

Reply:
Isto što i gore. Puca mi bez toga.